packages/lesswrong/components/search/SearchPage.tsx

```diff
@@ -110,40 +127,60 @@ const SearchPage = ({classes}:{
       </div>
       <CurrentRefinements />
       <div className={classes.columns}>
-      <Components.ErrorBoundary>
+      <ErrorBoundary>
+        <div className={classes.usersList}>
+          <Index indexName={algoliaIndexNames.Users}>
+            <div className={classes.header}>
+              <Typography variant="body1">
+                Users
+              </Typography>
+              <SearchPagination />
+            </div>
+            <Configure hitsPerPage={6} />
+            <Hits hitComponent={(props) => <UsersSearchHit {...props} />} />
+          </Index>
+        </div>
+      </ErrorBoundary>
+      <ErrorBoundary>
           <div className={classes.searchList}>
             <Index indexName={algoliaIndexNames.Posts}>
               <div className={classes.header}>
-                <Typography variant="body1">Posts</Typography>
+                <Typography variant="body1">
+                  Posts
+                </Typography>
                 <SearchPagination />
               </div>
 
-              <Configure hitsPerPage={10} />
+              <Configure hitsPerPage={6} />
```

Comment:
This is the change I'm most hesitant about. I'm trying it out now, but my anticipation is that this is likely something that makes the small-screen experience better and the large-screen experience worse.

Reply:
I futzed around a bit with some options to make it more variable. I wasn't sure what the best solution was. There are some obvious "make it scale based on monitor height" options, but I don't know if those are actually best.

It definitely needs to be this low on mobile, and probably should be this low at my current resolution. I think I'd prefer if it worked more like a "load more" than a pagination, where the load more loads a bunch more at once.

Reply:
Yeah, I think I don't like the 10->6 change. It only goes half-way down my screen.

Reply:
Seems like partially an artifact of the Next buttons being at the top rather than the bottom.

Reply:
On mobile, I think the Users and Tags sections should actually have *less* than 6; it is very unusual to be looking for a user or tag and have it be low-down in the search results.